browser-interface/packages/unity-interface/loader.ts

```diff
@@ -128,6 +137,24 @@ async function initializeWebRenderer(options: RendererOptions): Promise<Decentra
     // This function is called from the unity renderer to send messages back to the scenes
     BinaryMessageFromEngine(data: Uint8Array) {
       if (!!onBinaryMessage) onBinaryMessage(data)
+    },
+
+    RequestFromEngine(request: { type: string, payload: any }) {
+      const requestType = request.type as EngineRequestsNames
+      return onRequest(requestType, request.payload)
+
+      function onRequest<T extends EngineRequestsNames>(type: T, payload: IEngineRequests[T]) {
```

Comment:
why we have a fn here instead of running the code ?

Reply:
no reason at all, I was testing something al left the code like that :P
changed to remove that function